### ISSUE TITLE
Using knockoutjs with BForms

### ISSUE THREAD

**Issue by SyntaxThemes** [vtfuture/BForms#265]
Hi

Is anyone using knockoutjs with the BForms components.  Its seems ok with textbox etc but I am having a problem with the following control

@Html.BsDateTimePickerFor( m => m.StartDate, new { data_bind = "value: StartDate" } )

The binding will not working and even when I change to StartDate data_bind = "value: StartDate.DateValue  This also does not work.

Has anyone tried using these controls with knockoutjs?

Thanks

**Comment by alessandromuresan** [comment]
@SyntaxThemes 

Yeah, BForms wasn't exactly created with Knockout in mind, so you may experience some compatibility isues with more complex components, such as the Datepicker. I'll try and set up an example for myself and see what the problem could be, later today. Could you please copy-paste your client-side code here (including your Knockout view-model)?

Regards,
Alessandro

**Comment by SyntaxThemes** [comment]
Thanks for your response.

When you use the data_bind to create a jquery data parameter, this value is placed on the TextValue input of the datetime control instead of the DateValue part.

<input class="form-control bs-datetime" id="User_LastContacted_TextValue" name="User.LastContacted.TextValue" type="" value="" data-bind="{value: LastContacted}">
<input class="bs-date-iso" data-for="User.LastContacted.TextValue" id="User_LastContacted_DateValue" name="User.LastContacted.DateValue" type="hidden" value="">
